### DISCUSSION TITLE
How to turn off icon labeling on task bar

### DISCUSSION THREAD

**Discussion by gordom** [valinet/ExplorerPatcher#3559] [Q&A]
Hello there.
Can I turn off these icon labels on task bar?
![111a](https://github.com/user-attachments/assets/e5a8b96b-d761-4f1e-bbf9-e26c399b86f0)
I would like to get rid of the above text, keeping icons ungrouped and displayed in 2 lines. Any help will be appreciated. Regards.

**Comment by pyrates999** [comment]
Install windhawk and install this mod for it: https://windhawk.net/mods/taskbar-labels

**Reply by gordom** [reply]
I gave a try to this mode, but it doesn't seem to be helpful. I set a font size to 0, I left "Label for multiple items" and "Label for a single item" fields blank: none of these settings force system **to hide** labels in my case. Maybe I'm doing something wrong.

**Reply by pyrates999** [reply]
you need to kill the process explorer.exe and run explorer.exe

**Reply by gordom** [reply]
Yes, I did that just after installing and setting up this mod. As I said, I tried several settings. Each time I made any adjustment in the mod, explorer.exe was restarted. The problem still persists.

**Reply by milestprower92** [reply]
> Yes, I did that just after installing and setting up this mod. As I said, I tried several settings. Each time I made any adjustment in the mod, explorer.exe was restarted. The problem still persists.

Open search and type Properties (ExplorerPatcher) then click on "Combine primary icons on the taskbar" and choose "Always Combine"

**Reply by milestprower92** [reply]
> Install windhawk and install this mod for it: https://windhawk.net/mods/taskbar-labels

@pyrates999 that mod modifies text labels on the **Windows 11** Taskbar not the WIndows 10 one

**Reply by pyrates999** [reply]
thanks for letting me know @milesprower2293 :)

They don't want it combined though.  They want it uncombined, but without the text.

**Reply by gordom** [reply]
> > Yes, I did that just after installing and setting up this mod. As I said, I tried several settings. Each time I made any adjustment in the mod, explorer.exe was restarted. The problem still persists.
> 
> Open search and type Properties (ExplorerPatcher) then click on "Combine primary icons on the taskbar" and choose "Always Combine"

My settings: Combine icons on the taskbar and hide labels > Always. The problem still persists.

**Reply by pyrates999** [reply]
Is there a reason why you don't want to use the windows 10 taskbar?

**Reply by gordom** [reply]
That's exactly what I do - I use the Windows 10 taskbar to have a 2 or 3 line taskbar. And I want to disable the labels (there is a problem with it) having icons ungrouped at the same time (I achieved that with a help of windhawk mode).

**Reply by pyrates999** [reply]
Since you haven't provided this yet:

Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Reply by milestprower92** [reply]
> That's exactly what I do - I use the Windows 10 taskbar to have a 2 or 3 line taskbar. And I want to disable the labels (there is a problem with it) having icons ungrouped at the same time (I achieved that with a help of windhawk mode).

you can use 7+ taskbar tweaker if you are using the "Windows 10" Taskbar style

**Reply by gordom** [reply]
> Since you haven't provided this yet:
> 
> Please post windows version and build. You can find this by running winver. Please include if you are running x86-64 or Arm. Please post explorer patcher version.
> 
> Please include what settings you changed in explorer patcher properties application. These should be screen shots. If you can, please change it to english first.
> 
> Please include any other software you have installed that modifies the UI of windows.

I'm running Windows 11 ver. 23H2 (build 22631.4037). The architecture is x86-64. 
Explorer Patcher ver. 22621.3527.65.5

![Image 001](https://github.com/user-attachments/assets/ce3059cb-bcfc-4ce2-9934-ccfee8e7991a)
![Image 002](https://github.com/user-attachments/assets/2614ba45-9169-4e11-8b70-81f670682bf3)
![Image 003](https://github.com/user-attachments/assets/42a6f8af-1fad-4821-ab45-44ee693b8896)
![Image 004](https://github.com/user-attachments/assets/8792d45d-cd08-4d46-b8b3-d239b0034c41)
![Image 005](https://github.com/user-attachments/assets/c25415df-ab06-47c3-b188-268d9b8decfd)
![Image 006](https://github.com/user-attachments/assets/15f8e571-d31b-41df-9d5c-15d12d998cb4)


I'm also using Windhawk v1.5.1 with the following mods being installed:
![Windhawk](https://github.com/user-attachments/assets/9a5a4127-8708-4440-9c4a-5f0d248ae4d3)

The taskbar looks like the following screen shot:
![TaskBar](https://github.com/user-attachments/assets/dc3c1364-4eee-408a-ad87-a0d3a688e408)

And my goal is to hide labels.

**Reply by pyrates999** [reply]
Since you're using a windows 10 taskbar, install 7+ Taskbar Tweaker: https://ramensoftware.com/7-taskbar-tweaker

It has an option to set icons to not be combined and to not label them.

**Reply by gordom** [reply]
> Since you're using a windows 10 taskbar, install 7+ Taskbar Tweaker: https://ramensoftware.com/7-taskbar-tweaker
> 
> It has an option to set icons to not be combined and to not label them.

After installing 7+ Taskbar Tweaker the problem has not been resolved. I uninstalled Windhawk, and reinstalled Explorer Patcher. The labels are still there. I would like to go through the installation process again from the scratch, but I don't know if Explorer Patcher is being fully uninstalled together with the all settings. Can I be sure of that? If I chose restore default settings option, I'm getting an error prompt saying that not all settings can be restored. I was trying to do so on both, standard user and admin account. There is no difference.

**Reply by pyrates999** [reply]
EP doesn't run on a standard user account.  It requires administrative privileges to run.

You can run EP, windhawk, and 7+ Taskbar Tweaker together.

**Reply by pyrates999** [reply]
You install EP and set it to use the windows 10 taskbar.  Then after installing 7+ Taskbar Tweaker, you still need to enable it:

![image](https://github.com/user-attachments/assets/1599d5ed-94e7-4b2a-8707-69f5bd0cd2ea)

**Reply by gordom** [reply]
> EP doesn't run on a standard user account. It requires administrative privileges to run.
> 
> You can run EP, windhawk, and 7+ Taskbar Tweaker together.

Well, it seems I managed to set the taskbar up finally. At least for that moment, it behaves as I want. It doesn't matter if I use standard user account or admin. First, I set Windows taskbar setting Combine (group) icons and hide labels on taskbar to Always. Then I set ExplorerPatcher taskbar options:
taskbar style: Windows 10
combine taskbar icons on primary taskbar: Always combine > against default setting

In the end, I set 7+ Taskbar Tweaker Grouping and combining section to Don't group.
And the result in my Windows 11 is like this:
![nowy taskbar](https://github.com/user-attachments/assets/459ae42b-9fcc-4e0a-a9aa-2dfde6eae999)
Thank you @pyrates999 and @milesprower2293  very much for all your help, time and effort.

EDIT: There is one small disadvantage, though. All active programs icons are highlighted by the blue underline.

**Reply by pyrates999** [reply]
The blue underline is supposed to be there to indicate the application is open.

**Reply by gordom** [reply]
> The blue underline is supposed to be there to indicate the application is open.

OK, I understand. Thank you.
